src/screens/FeedScreen/elements/Search.js

```diff
@@ -47,12 +47,18 @@ const Search = ({
 };
 
 export const HeaderStyles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    backgroundColor: COLORS.white,
+    marginBottom: SIZES.base,
+    marginHorizontal: SIZES.base
+  },
   searchContainer: {
     flex: 1
   },
   wrapperSearch: {
     flex: 1,
-    backgroundColor: '#F5F5F5',
+    backgroundColor: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are primarily related to the color scheme and layout of the application. The `backgroundColor` property in the `wrapperSearch` style has been updated from a hardcoded value `'#F5F5F5'` to `COLORS.lightgrey`, which is more maintainable as it allows for centralized control over color schemes. 

However, I noticed that the `container` style was added but not used anywhere in this file. If it's not being used, it would be better to remove it to avoid confusion and keep the codebase clean.

```diff
  export const HeaderStyles = StyleSheet.create({
-  container: {
-    flexDirection: 'row',
-    backgroundColor: COLORS.white,
-    marginBottom: SIZES.base,
-    marginHorizontal: SIZES.base
-  },
   searchContainer: {
     flex: 1
   },
   wrapperSearch: {
     flex: 1,
     backgroundColor: COLORS.lightgrey,
     marginLeft: 20,
     marginRight: 20,
     borderRadius: 8,
```
If it's intended to be used elsewhere, please disregard this comment.


<!-- This is an auto-generated comment by OSS CodeRabbit -->